downloader/run

```diff
@@ -2,12 +2,34 @@
 #coding:utf-8
```

Comment:
this file doesn't have a file extension, is that intentional?

Reply:
Yes, because of the shebang at the top and the execute bit set, this will run like `./run` so removing the .rb is on purpose so folks don't think they need to run it with `ruby`

Reply:
ahh TIL